index.Rmd

```diff
@@ -365,9 +365,9 @@ allyears= transform(allyears,
                     pctProfjobs_c=Profjobs_c/jobsTotal_c*100,
                     pctPoverty_t=Poverty_t/PovertyTotal_t*100,
                     pctPoverty_c=Poverty_c/PovertyTotal_c*100)
-
-# head(allyears)
```

Comment:
I'm curious if you just used this for our own learning/organization. If so, you might want to exclude from the final code.